plugins/queue/scheduler.js

```diff
@@ -538,22 +539,19 @@ async function init(executor) {
     executor.multiWorker.on('reEnqueue', (workerId, queue, job, plugin) =>
         logger.info(`worker[${workerId}] reEnqueue job (${plugin}) ${queue} ${JSON.stringify(job)}`)
     );
-    executor.multiWorker.on('success', (workerId, queue, job, result) =>
-        logger.info(`worker[${workerId}] job success ${queue} ${JSON.stringify(job)} >> ${result}`)
+    executor.multiWorker.on('success', (workerId, queue, job, result, duration) =>
+        logger.info(`worker[${workerId}] job success ${queue} ${JSON.stringify(job)} >> ${result} (${duration}ms)`)
     );
-    executor.multiWorker.on('failure', (workerId, queue, job, failure) =>
-        logger.info(`worker[${workerId}] job failure ${queue} ${JSON.stringify(job)} >> ${failure}`)
+    executor.multiWorker.on('failure', (workerId, queue, job, failure, duration) =>
+        logger.info(`worker[${workerId}] job failure ${queue} ${JSON.stringify(job)} >> ${failure} (${duration}ms)`)
     );
     executor.multiWorker.on('error', (workerId, queue, job, error) =>
         logger.error(`worker[${workerId}] error ${queue} ${JSON.stringify(job)} >> ${error}`)
     );
 
-    // multiWorker emitters
-    executor.multiWorker.on('internalError', error => logger.error(error));
-
     executor.scheduler.on('start', () => logger.info('scheduler started'));
     executor.scheduler.on('end', () => logger.info('scheduler ended'));
-    executor.scheduler.on('master', state => logger.info(`scheduler became master ${state}`));
```

Comment:
renamed to `leader` in 7.0.0
https://github.com/actionhero/node-resque/releases/tag/v7.0.0